src/res/translation/translation_nl_NL.ts.bak

```diff
@@ -0,0 +1,3492 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This was probably created by your editor and should not be committed. :)

Reply:
Apology for the rookie mistake....already removed from the PR